controller/backing_image_controller.go

```diff
@@ -294,15 +294,15 @@ func (bic *BackingImageController) cleanupBackingImageManagers(bi *longhorn.Back
 		if bim.DeletionTimestamp != nil {
 			continue
 		}
-
+		bimLog := log.WithField("backingImageManager", bim.Name)
 		// Directly clean up old backing image managers (including incompatible managers).
 		// New backing image managers can detect and reuse the existing backing image files if necessary.
 		if bim.Spec.Image != defaultImage {
-			log.WithField("backingImageManager", bim.Name).Info("Start to delete old backing image manager")
+			bimLog.Info("Start to delete old/non-default backing image manager")
```

Comment:
nit: "Start to delete old/non-default ...."

Reply:
Generally, when making logs around an action can follow some logging conventions which can help us easily understand the context. 

- "Verb+ing ..." for before taking the action
- "Failed to Verb ..." for failing the action
- "Verb+ed" for taking the action successfully (this is optional)

However, this is just a Nit.